src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+	public String value;
+    protected boolean isPrivate;
+    
+    @Override
```

Comment:
- i notice that you are not defining/using a super class constructor. 
- think how you can implement a super class constructor and push data validation and assignment to the parent class.